### PR TITLE
Untracking files previously committed from new .gitignore

### PR DESCRIPTION
**Pull request by aaronTHJ**
Wiki article addressing issue #367

### CODE REVIEW

File: Ignore-Files-Committed-To-Git-Repo.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Untrack files previously committed from new .gitignore
+
+To untrack a <i>single</i> file, ie stop tracking the file but not delete it from the system use:
```
Comment (raisedadead) [inline Ignore-Files-Committed-To-Git-Repo.md]: Use [markdown](https://guides.github.com/features/mastering-markdown) instead.
`<i>single<i>` to `_single_`

File: Ignore-Files-Committed-To-Git-Repo.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+```git rm --cached filename```
+
+To untrack <i>every</i> file in .gitignore:
```
Comment (raisedadead) [inline Ignore-Files-Committed-To-Git-Repo.md]: Use [markdown](https://guides.github.com/features/mastering-markdown) instead.
`<i>single<i>` to `_single_`

**Comment by raisedadead** [comment]
Once you have done that please [Squash you commits](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one)

Let us know in the [Help Chat room](https://gitter.im/FreeCodeCamp/Wiki) if you need any help.

File: Ignore-Files-Committed-To-Git-Repo.md (hunk 1 above)
Comment (alayek) [inline Ignore-Files-Committed-To-Git-Repo.md]: ie should be i.e.

**Pull request closed without merging**